### DISCUSSION TITLE
Option --no-warn don't suppress all warnings checking a PDF

### DISCUSSION THREAD

**Discussion by vf1962** [qpdf/qpdf#850] [General]
Maybe I'm using parameters in a wrong way but executing this qpdf
**QPDF.EXE** **--no-warn**  **--check** **"C:\temp\Outlook.Dump\2022.11.23 pulizia - redmine #31190 ESEGUITA.pdf"**

The result is
checking C:\temp\Outlook.Dump\2022.11.23 pulizia - redmine #31190 ESEGUITA.pdf
PDF Version: 1.4
File is not encrypted
File is linearized
WARNING: shared object 0 length mismatch: hint table = 65536; computed = 187
WARNING: shared object 1 length mismatch: hint table = 65536; computed = 13879
WARNING: shared object 2 length mismatch: hint table = 65536; computed = 21
                                                  _[OMISSIS]_
WARNING: shared object 239 length mismatch: hint table = 65536; computed = 35
WARNING: shared object 240 length mismatch: hint table = 65536; computed = 33
WARNING: shared object 241 length mismatch: hint table = 65536; computed = 924
WARNING: object count mismatch for page 0: hint table = 0; computed = 242

The PDF file is a Chrome print-out from Redmine web application.
Tank you

**Comment by jberkenbilt** [comment]
What version of qpdf are you using? Some older versions of qpdf leaked linearization warnings when `--no-warn` was given, but I'm pretty sure this has been fixed.

**Reply by jberkenbilt** [reply]
I take it back -- current qpdf still gives linearization warnings with `--no-warn`.

**Comment by jberkenbilt** [comment]
Closing in favor of #851
